### DISCUSSION TITLE
article/notion-next-giscus

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#93] [Announcements]
# article/notion-next-giscus

这是一个由NotionNext生成的站点

https://docs.tangly1024.com/article/notion-next-giscus

**Comment by Max7C** [comment]
搞的头都大了……

**Reply by Max7C** [reply]
我在Gitalk上卡了好多天，最终放弃了改成Giscus了。稍后准备再弄个Twikoo就不搞了。欢迎随便逛逛我的主页：https://52cnb.top/，对就是我爱吹牛逼。

**Reply by blackcat1402** [reply]
感觉Giscus就很棒了，操作也简单

**Comment by Lrcx** [comment]
这个类似github的issue评论
